service/matching/matching_engine_test.go

```diff
@@ -211,6 +212,7 @@ func (s *matchingEngineSuite) newConfig() *Config {
 	} else if s.newMatcher {
 		useNewMatcher(res)
 	}
+	res.AutoEnableV2 = func(_, _ string, _ enumspb.TaskQueueType) bool { return true }
```

Comment:
I'd just put this in defaultTestConfig, no? it won't affect any of the other tests that use that.

also we have helpers for those:

```suggestion
	res.AutoEnableV2 = dynamicconfig.GetBoolPropertyFnFilteredByTaskQueue(true)
```